### ISSUE TITLE
CSS Live Update not working anymore

### ISSUE THREAD

**Issue by ThomasH99** [codenameone/CodenameOne#3356]
As discussed in the comments (Jan/Feb 2021) here: https://www.codenameone.com/blog/live-css-update.html#comment-24398.

The live update stopped working some time ago, but works for a HelloWorld project. The css file is read correctly when I compile the app, here's the compile log: 

```
JavaFX is loaded
Retina Scale: 2.0
Updating merge file /Users/user/NetBeansProjects/myapp/css/theme.css.merged
Input: /Users/user/NetBeansProjects/myapp/css/theme.css
Output: /Users/user/NetBeansProjects/myapp/src/theme.res
Acquiring lock on CSS checksums file /Users/user/NetBeansProjects/myapp/.cn1_css_checksums...
Lock obtained
File has not changed since last compile.
Releasing lock
CSS file successfully compiled.  /Users/user/NetBeansProjects/myapp/src/theme.res
```

When I modify and save the css file while the simulator is running, there is nothing at all displayed in the log. 

Here's the codenameone_settings.properties as asked by Steve (I've deleted all info related to certificates and passwords): 

```
android.playService.ads=true
codename1.android.keystoreAlias=myapp
codename1.arg.android.debug=false
codename1.arg.android.multidex=true
codename1.arg.android.release=true
codename1.arg.desktop.theme=iOS7Theme
codename1.arg.ios.application_exits=false
codename1.arg.ios.background_modes=fetch
codename1.arg.ios.enableBadgeClear=false
codename1.arg.ios.includePush=true
codename1.arg.ios.interface_orientation=UIInterfaceOrientationPortrait\:UIInterfaceOrientationPortraitUpsideDown\:UIInterfaceOrientationLandscapeLeft\:UIInterfaceOrientationLandscapeRight
codename1.arg.ios.multitasking=true
codename1.arg.ios.newStorageLocation=true
codename1.arg.ios.project_type=ios
codename1.arg.ios.statusbar_hidden=false
codename1.arg.ios.testFlight=true
codename1.arg.java.version=8
codename1.cssTheme=true
codename1.displayName=myapp
codename1.icon=icon.png
codename1.mainName=myapp
codename1.packageName=com.myapp.myapp
codename1.vendor=myapp
codename1.version=0.10
csswatcher.enabled=true
libVersion=201
```

**Comment by shannah** [comment]
Just compiling relevant details from the blog comment thread:

IDE: NetBeans 12.2
OS: Mac

A Hello World project on same machine with same IDE works (i.e. css auto update works on that), but this project does not work.

Some standard troubleshooting to try:

1. Delete the src/theme.res file
2. Delete the "css/*.merged" file.

**Comment by ThomasH99** [comment]
I already tried deleting the css/*.merged. I now tried that together with deleting the src/theme.res, but still not working. Is there maybe a log file somewhere that might help understand what happens? 

It may also be  important to note that the generation *does* work when I compile the app, just not when I change the css file while the simulator is running. 

An additional data point is that the generated theme.res is quite big (372K). However it's been roughly that size for a long time and it worked nevertheless, so it seems unlikely to be the reason (unless I suddenly hit some kind of memory limit).

**Comment by ThomasH99** [comment]
Thanks Steve, I just pulled the updated CN1 code and now it works. 

Btw, after I had reported this issue, I noticed another possibly issue: I believe that in the past, if you opened ComponentInspector with the Simulator, and then build and run another simulator it would automatically open the ComponentInspector again - this was quite handy as it saves several clicks. Before I saw your fix, I thought the two issues might be related, but I just checked and even with the latest sources, ComponentInspector doesn't open automatically. Let me know if this is a bug, then I can create a separate issue on it.
